Properties/AssemblyInfo.cs

```diff
@@ -32,4 +32,3 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.3")]
-[assembly: AssemblyFileVersion("1.0.0.3")]
```

Comment:
`AssemblyFileVersion` defaults to `AssemblyVersion` if missing anyway, redundant for us.